packages/fx-core/src/plugins/resource/bot/dotnet/plugin.ts

```diff
@@ -51,6 +53,31 @@ export class DotnetBotImpl extends TeamsBotImpl {
     return ResultFactory.Success(result);
   }
 
+  public async postLocalDebug(context: PluginContext): Promise<FxResult> {
+    await super.postLocalDebug(context);
+    const appSettingsPath = path.join(context.root, PathInfo.appSettingDevelopment);
+    try {
+      let appSettings = await fs.readFile(appSettingsPath, "utf-8");
+      const botId = context.envInfo.state.get(PluginNames.BOT)?.get(DependentPluginInfo.botId);
```

Comment:
I wonder how does this code know to get botId and Bot password for local settings?